extra-files/galaxy-kickstart/galaxy-kickstart_tool_list.yml

```diff
@@ -0,0 +1,37 @@
+tools:
+- name: data_manager_fetch_genome_dbkeys_all_fasta
+  owner: devteam
+  revisions:
+  - 776bb1b478a0
+  tool_panel_section_label: null
```

Comment:
if you got this with the get_tool_list_from_gi.py script we should change it to skip null.